codalab/bundles/run_bundle.py

```diff
@@ -27,8 +27,8 @@ class RunBundle(DerivedBundle):
     METADATA_SPECS.append(MetadataSpec('request_time', basestring, 'Amount of time (e.g., 3, 3m, 3h, 3d) allowed for this run.', formatting='duration'))
     METADATA_SPECS.append(MetadataSpec('request_memory', basestring, 'Amount of memory (e.g., 3, 3k, 3m, 3g, 3t) allowed for this run.', formatting='size'))
     METADATA_SPECS.append(MetadataSpec('request_disk', basestring, 'Amount of disk space (e.g., 3, 3k, 3m, 3g, 3t) allowed for this run.', formatting='size'))
-    METADATA_SPECS.append(MetadataSpec('request_cpus', int, 'Number of CPUs allowed for this run.'))
-    METADATA_SPECS.append(MetadataSpec('request_gpus', int, 'Number of GPUs allowed for this run.'))
+    METADATA_SPECS.append(MetadataSpec('request_cpus', int, 'Number of CPUs allowed for this run.', default=1))
```

Comment:
Is this the place for defaults?  It seems strange that defaults would only be available for the cpu and gpu flags when some sort of defaults are of course in place for the other fields...

Reply:
All MetadataSpecs have a default default which is the default constructor for the spec's data type, unless overridden by a custom default value, which we need for request_cpus somewhere